cmd/main.go

```diff
@@ -32,54 +33,118 @@ func main() {
 		panic(err)
 	}
 
-	base := usecases.DeploymentUseCase{
+	eventsLogger, err := providers.CreateWalWriter(filepath.Join(config.AppConfig.DataDir, constants.EventLogDir))
+	if err != nil {
+		panic(err)
+	}
+
+	accessLogger, err := providers.CreateWalWriter(filepath.Join(config.AppConfig.DataDir, constants.AccessLogDir))
+	if err != nil {
+		panic(err)
+	}
+
+	e := providers.NewEntropy([]byte(config.AppConfig.InitialHash))
```

Comment:
`e` variable name sounds confusing, please rename it to `entropy`
```suggestion
	entropy := providers.NewEntropy([]byte(config.AppConfig.InitialHash))
```